.ci/test_daemon.py

```diff
@@ -356,6 +410,20 @@ def main():
     print('Running the `MultiplyAddWorkChain`')
     run_multiply_add_workchain()
 
+    # Submitting the calcfunction through the launchers
+    print(f'Submitting {NUMBER_CALCULATIONS} calcfunction to the daemon')
+    for counter in range(1, NUMBER_CALCULATIONS + 1):
+        inputval = counter
+        proc, expected_result = launch_calcfunction(counter=counter, inputval=inputval)
+        expected_results_calcfunctions[proc.pk] = expected_result
+
+    # Submitting the workfunction through the launchers
+    print(f'Submitting {NUMBER_CALCULATIONS} workfunction to the daemon')
+    for counter in range(1, NUMBER_CALCULATIONS + 1):
+        inputval = counter
+        proc, expected_result = launch_workfunction(counter=counter, inputval=inputval)
+        expected_results_workfunctions[proc.pk] = expected_result
```

Comment:
I think launching just one of each here is fine. Don't think running multiple process functions tests anything of additional value